client/scss/grids/_grid_base.scss

```diff
@@ -13,6 +13,72 @@
   }
 }
 
+.grid--scroll {
+  position: relative;
+  padding-top: $vertical-space-unit;
+  padding-bottom: 4 * $vertical-space-unit;
+  padding-left: map-get($container-padding, 'small');
+  padding-right: map-get($container-padding, 'small');
+
+  @include respond-to('medium') {
+    padding-bottom: 6 * $vertical-space-unit;
+    padding-left: map-get($container-padding, 'medium');
+    padding-right: map-get($container-padding, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding-left: map-get($container-padding, 'large');
+    padding-right: map-get($container-padding, 'large');
+  }
+
+  @include respond-between('small', 'medium') {
+    // This leaves a comfortable amount of the 'next' grid
+    // item displaying in this configuration
+    // TODO: calculate this based on number of contained grid__cells
+    width: 250%;
```

Comment:
I'm not sure if [this](https://jsfiddle.net/jamesgorrie/og2khnxw/) solves what you're talking about - but it seemed an easy way to use flex (and `flex-shrink: 0`) to let the container sort it's own width out.